library/libGUI/lib_process.ks

```diff
@@ -0,0 +1,81 @@
+@lazyglobal off.
+
+run lib_exec.
+
+function process_finished{
```

Comment:
It doesn't really matter, in stock kOS we often have suffixes of just verb in past tense without "is", such as ship:loaded.